Std/Data/BitVec/Basic.lean

```diff
@@ -53,7 +53,7 @@ protected def toNat (a : BitVec n) : Nat := a.toFin.val
 /-- Return the `i`-th least significant bit. -/
 @[inline] def getLsb (x : BitVec w) (i : Nat) : Bool := x.toNat &&& (1 <<< i) != 0
 
-/-- Return the `i`-th least significant bit. -/
+/-- Return the `i`-th most significant bit. -/
```

Comment:
This implementation returns the lsb when `i` is out of range, while getLsb returns 0 when `i` is out of range. I suggest making them both return zero and document it as such.